config/environments/development.rb

```diff
@@ -5,11 +5,11 @@
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.paperclip_defaults = {
-      :storage => :s3,
-      :s3_credentials => {
-        :bucket => ENV['S3_BUCKET_NAME'],
-        :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-        :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+    :storage => :s3,
+    :s3_credentials => {
+      :bucket => ENV['beerworld'],
+      :access_key_id => ENV['AKIAI6YTIHHF2PMSYMRQ'],
+      :secret_access_key => ENV['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']
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.